Trfgen/src/trfgen.c

```diff
@@ -368,6 +377,11 @@ void trfgen_start_test(int test_id, char *host_ip, char *bind_ip, char *host_por
          iperf_set_test_bind_address(test, bind_ip);
          iperf_set_test_server_port( test, port );
          iperf_set_test_duration( test, tstcfg[test_id].duration);
+         // Add delay to make sure that ipv6 addr is configured on eth2 i/f
+         if (ip_version == 6) {
+           printf("Sleeping for 5 secs\n");
```

Comment:
Better to have a check that interface is configured here and proceed only when the interface is configured. This would reduce flakiness.

Reply:
Addressed